### ISSUE TITLE
Stage_one WARNING: Duplicated sequences

### ISSUE THREAD

**Issue by HuangLoong** [xinehc/args_oap#58]
I generated a warning message while running ARGs OAP stage_one on the server. The metagenomic data is sourced from NCBI and has gone through the kneaddata process. However, when I used my own sequencing data, no warning message was generated. What is the reason and will it affect subsequent analysis? Thanks very much for your answer！
WARNING: Duplicated sequences in sequence extraction.
WARNING: Duplicated sequences in 16S copy number calculation.
WARNING: Duplicated sequences in cell number calculation.

![image](https://github.com/xinehc/args_oap/assets/162462313/72878107-826e-4a94-9a61-f4d0e9af6037)

![image](https://github.com/xinehc/args_oap/assets/162462313/c73df1fc-9e9a-478b-8129-59b941081ed5)

**Comment by HuangLoong** [comment]
![6hnr4hw7](https://github.com/xinehc/args_oap/assets/162462313/bb8ffb2a-5b37-4641-b211-c124909c9dd4)
![qi87u79o](https://github.com/xinehc/args_oap/assets/162462313/bcec23b0-d00b-4f04-92a8-3ce4ae9da9a6)

**Comment by xinehc** [comment]
This likely means some of your sequences have exactly identical sequence id. You can check the duplication using e.g. `seqkit rmdup`.

**Comment by HuangLoong** [comment]
> This likely means some of your sequences have exactly identical sequence id. You can check the duplication using e.g. `seqkit rmdup`.

We renamed the sequence after downloading the raw data from NCBI, and no duplicate sequence IDs were detected using seqkit rmdup.

**Comment by xinehc** [comment]
Hi,

do you mind sharing a minimal reproducible example? Does your _1 and _2 files have identical sequence id?

```
cat _1.fa _2.fa > seq.fa
seqkit rmdup seq.fa -D dup.fa
```

**Comment by HuangLoong** [comment]
> Hi,
> 
> do you mind sharing a minimal reproducible example? Does your _1 and _2 files have identical sequence id?
> 
> ```
> cat _1.fa _2.fa > seq.fa
> seqkit rmdup seq.fa -D dup.fa
> ```

Thank you. Based on the command you provided, I did detect some duplicated sequences. I then extracted the previously retrieved duplicate IDs from the paired-end sequencing file. Duplicate IDs were found only in one of the pairwise sequence files. I wonder if this error will affect the final result output.
![image](https://github.com/xinehc/args_oap/assets/162462313/79338eba-62dd-4033-a089-7ddc78a05cca)
